lib/imapoptions

```diff
@@ -2129,14 +2173,26 @@ If all partitions are over that limit, this feature is not used anymore.
    conversations quota counts, which count multiple copies of exactly the
    same message (by GUID) as only one */
 
-{ "quotawarn", 90, INT, "2.3.17" }
+{ "quotawarn", 90, INT, "UNRELEASED", "UNRELEASED", "quotawarnpercent" }
+/* Deprecated in favour of \fIquotawarnpercent\fR. */
+
+{ "quotawarnpercent", 90, INT, "UNRELEASED" }
 /* The percent of quota utilization over which the server generates
    warnings. */
 
-{ "quotawarnkb", 0, INT, "2.3.17" }
-/* The maximum amount of free space (in kB) at which to give a quota
-   warning (if this value is 0, or if the quota is smaller than this
-   amount, then warnings are always given). */
+{ "quotawarnkb", NULL, BYTESIZE, "UNRELEASED", "UNRELEASED", "quotawarnsize" }
+/* Deprecated in favour of \fIquotawarnsize\fR. */
+
+{ "quotawarnsize", "0", BYTESIZE, "UNRELEASED" }
+/* The maximum amount of free space at which to give a quota warning
+   (if this value is 0, or if the quota is smaller than this
+   amount, then warnings are always given).
+.PP
+   Note that quota has kibibyte granularity.  Values specified here will
+   be truncated to the nearest whole kibibyte.
+.PP
+   For backward compatibility, if no unit is specified, kibibytes is
+   assumed.  */
```

Comment:
There is no need for backword compatibility, since the option `qoutawarnsize` has not existed in old versions.